### PR TITLE
Add Mount State sensor to the Quest

### PR DESCRIPTION
**Pull request by SkechyWolf**
## Summary

Adds a mount state sensor to the quest
Shows as a binary sensor on the frontend 

## Screenshots

image

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#659

## Any other notes

N/A

### CODE REVIEW

File: app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt [hunk 1]
```diff
@@ -50,6 +50,7 @@ class SensorReceiver : SensorReceiverBase() {
             PowerSensorManager(),
             PressureSensorManager(),
             ProximitySensorManager(),
+            QuestSensorManager(),
```
Comment (dshokouhi) [inline app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt:53]: We need to also add the intent action string and sensor ID to the `skippableActions` below

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Great work the sensor is reporting as expected when I tested the debug APK! Just 1 minor comment so we can skip unwanted updates if the user did not want it enabled.

File: app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt (hunk 1 above)
Comment (SkechyWolf) [reply app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt:53]: Should be fixed

**Review by dshokouhi — APPROVED** [review]

**Comment by dshokouhi** [comment]
Just noticed the docs link goes to an already merged PR, lets update the docs to mention this new sensor on the Quest specific page please.

**Review by JBassett — APPROVED** [review]

**Pull request merged**